packages/internal-playwright-helpers/package.json

```diff
@@ -8,7 +8,8 @@
     "access": "public"
   },
   "scripts": {
-    "build": "rollup -c rollup.config.mjs"
+    "build": "rollup -c rollup.config.mjs",
+    "clean": "rimraf dist"
```

Comment:
As a general rule of thumb, any time you have a script that builds output, it's wise to also include a script to clean up that output, such that you can always easily start from a fresh & known state, and that one build doesn't bleed into the other on your development machine